public/src/app-bar.html

```diff
@@ -115,8 +115,17 @@
         });
       }
 
-      _refreshBalance(reg) {
-        this.balance = "ℂ" + (reg.status.userBalance || 0).toFixed(3);
+      _refreshBalance(reg, fetched) {
+        setInterval(() => {
+          this.balance = "ℂ" + this._truncateBalance($core.balance);
+        }, 1000);
+      }
+
+      _truncateBalance(value) {
+        if (!value) {
+          return 0;
+        }
+        return (Math.floor(value * 1000) / 1000).toFixed(3);
       }
```

Comment:
This will always produce .000 
You may want to remove the floor call.

Reply:
10.123456 * 1000 = 101234.56
Math.floor(101234.56) = 10123
10123 / 1000 = 10.123
10.123.toFixed(3) = 10.123

Am I missing something?

Reply:
I misread the parenthesis. all good.